README.md

```diff
@@ -1,6 +1,8 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+## Making a change here to demonstrate how Sema works. 
```

Comment:
Please read through bubble sort notes:

Bubble sort, sometimes referred to as sinking sort, is a simple sorting algorithm that repeatedly steps through the list to be sorted, compares each pair of adjacent items and swaps them if they are in the wrong order (ascending or descending arrangement). The pass through the list is repeated until no swaps are needed, which indicates that the list is sorted.



📄 [Oleksii Trekhleb](https://github.com/trekhleb/javascript-algorithms/tree/master/src/algorithms/sorting/bubble-sort#bubble-sort)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient